redfish-client/redfish-client

```diff
@@ -320,47 +344,55 @@ if __name__ == '__main__':
     logger.info("Arguments parsed")
     logger.debug(arguments)
 
-    # Get $HOME and $VIRTUAL_ENV environment variables.
+    # Get $HOME environment variables.
     HOME = os.getenv('HOME')
-    VIRTUAL_ENV = os.getenv('VIRTUAL_ENV')
 
-    if not HOME:
+    if(not HOME):
         print('$HOME environment variable not set, please check your system')
         logger.error('$HOME environment variable not set')
         sys.exit(1)
     logger.debug("Home directory : %s" % HOME)
-    
-    if VIRTUAL_ENV:
-        logger.debug("Virtual env : %s" % VIRTUAL_ENV)
-        
-    # Load master conf file
+
+    if(special_location()):
+        logger.debug("Location prefix: %s" % special_location())
+
+    # Load config
     config = ConfigParser.ConfigParser(allow_no_value=True)
-    logger.debug("Read master configuration file")
-    master_conf_file_path = "/etc/redfish-client.conf"
-    
-    if VIRTUAL_ENV:
-        logger.debug("Read master configuration file from virtual environment")
-        master_conf_file_path = VIRTUAL_ENV + master_conf_file_path
-        
-    if not os.path.isfile(master_conf_file_path):
-        print('Master configuration file not found at {}.'.format(master_conf_file_path))
-        logger.error('Master configuration file not found at %s.' % master_conf_file_path)
+    logger.debug("Read configuration file")
+    configfiles = ["/etc/redfish-client.conf",
+                   "/usr/local/etc/redfish-client.conf"]
```

Comment:
Humm again it's still hard coded. That should be outside of the code again for me.